packages/slate-editor/src/modules/editor-v4-void/VoidExtension.tsx

```diff
@@ -1,10 +1,14 @@
 import type { Extension } from '@prezly/slate-commons';
 import { EditorCommands } from '@prezly/slate-commons';
 import { isHotkey } from 'is-hotkey';
-import { Editor } from 'slate';
+import { Editor, Transforms } from 'slate';
+
+import { createParagraph } from '#modules/editor-v4-paragraphs';
 
 import { VOID_EXTENSION_ID } from './constants';
 
+import { isDeleting } from '#modules/editor-v4-image/lib';
```

Comment:
`isDeleting` should probably be lifted into the `#lib` folder, as it's shared between modules.